### PR TITLE
Species events data migrate and display

### PR DESCRIPTION
**Pull request by ya-web**

### CODE REVIEW

File: src/Command/EventsSpeciesGenerateCommand.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        /**
+         * @var Species[]
+         */
+        $allSpecies = $this->manager->getRepository(Species::class)->findAll();
+
+        if (empty($allSpecies)) {
```
Comment (kstefanini) [inline src/Command/EventsSpeciesGenerateCommand.php:40]: `$output->writeln('species not found');`

File: src/Command/EventsSpeciesGenerateCommand.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+            $output->writeln('Creating eventSpecies for species: '.$species->getVernacularName());
+
+            foreach ($eventsIds as $eventId) {
+                /**
+                 * @var Event
+                 */
+                $event = $eventRepository->find($eventId);
+
+                if (empty($event)) {
```
Comment (kstefanini) [inline src/Command/EventsSpeciesGenerateCommand.php:78]: `$output->writeln('event not found : '.$event);`

Comment (ya-web) [reply src/Command/EventsSpeciesGenerateCommand.php:78]: `$output->writeln(sprintf('event not found : %s stade %d.', $event->getName(), $event->getStadeBbch()));`

Comment (ya-web) [reply src/Command/EventsSpeciesGenerateCommand.php:78]: `$output->writeln(sprintf('event with id %d not found'.$eventId));`

**Pull request merged**